docusaurus.config.js

```diff
@@ -44,6 +44,13 @@ const config = {
               groupPathsBy: "tag",
             },
           },
+          ap_api: {
```

Comment:
We may want to get @briwylde08's input on the URL path used.

Reply:
What are we looking for here?

Reply:
It's a new path we will be using (will be https://developers.stellar.org/ap_api)
I'm guessing @JakeUrban wanted to double check if this looks good to you

Reply:
yep!